### PR TITLE
portals: increased delete security for brands

### PR DESCRIPTION
**Pull request by mmadariaga**
Applied for companies as well

Fixed error when multiple companies were selected

### CODE REVIEW

File: library/IvozProvider/Klear/Filter/OutgoingDDI.php [hunk 1]
```diff
@@ -10,8 +10,13 @@ public function setRouteDispatcher(KlearMatrix_Model_RouteDispatcher $routeDispa
         $pk = $routeDispatcher->getParam("pk", false);
 
         // Only display DDIs belonging to edited company
-        $this->_condition[] = "`companyId` = '" . $pk . "'";
+        if (is_array($pk)) {
+            //Avoid multiple choice error on klear
+            $this->_condition[] = "1=2";
```
Comment (Kaian) [inline library/IvozProvider/Klear/Filter/OutgoingDDI.php]: If there's no way to avoid the query, what about...
```
    $this->_condition[] = "FALSE";
```

Comment (mmadariaga) [inline library/IvozProvider/Klear/Filter/OutgoingDDI.php]: Umm, I like the fact that you can see that it's a condition itself easily

Comment (Kaian) [inline library/IvozProvider/Klear/Filter/OutgoingDDI.php]: It shouldn't. FALSE is a SQL keyword afaik

Comment (cruzccl) [inline library/IvozProvider/Klear/Filter/OutgoingDDI.php]: I don't understand the relationship between this feature and OutgoingDDI.php file :thinking:

Comment (mmadariaga) [inline library/IvozProvider/Klear/Filter/OutgoingDDI.php]: @Kaian I know, I just prefer it that way. It's kind of more universal. @cruzccl tiebreak this please :)

Comment (mmadariaga) [inline library/IvozProvider/Klear/Filter/OutgoingDDI.php]: @cruzccl Try to select a couple of companies and press delete button then

**Pull request merged**